### ISSUE TITLE
[Feature Request] Remember FOmod installer selected options

### ISSUE THREAD

**Issue by nikolay-borzov** [TanninOne/modorganizer#543]
Would be extremely useful when updating mods.
Data can be stored in meta.ini.
The only problem I see is how to determine that user is installing the same mod

**Comment by TanninOne** [comment]
I thought about something like this before, it would also be useful to provide "unattended" installs for stuff like step.

Unfortunately it would only work on xml fomods as I have limited control over the C# scripted installer.
Also, this could easily break if the installer changes. What if new options get introduced? What if the name of options changes even slightly? What if the order of options changes?

**Comment by nikolay-borzov** [comment]
I agree, issues will likely appear. Well, it will require more work on this feature.
I see it like this:

- I can me semi-automatic - only pre-select options
- MO saves each step selected options somewhere
- once installer run MO compares available options and selected options (saved)
    - If there a selected option isn't presented anymore - give user know (disabled text) that the option is no longer available
    - we could create kind of hash for the set of available options. If new installer's "hash" isn't equal to old one - show previously selected options somewhere in the installer window (so that user can refer to them and perform install)

Right it'll be a complex feature. At first we can simplify it in a following way:
- MO saves each step selected options somewhere
- On the installer dialog simply show selected options somewhere - so that user can refer to his old choices

A side note: Currently I just type selected options in a text file and then copy them to Notes of the mod
